statsmodels/robust/scale.py

```diff
@@ -49,6 +49,46 @@ def mad(a, c=Gaussian.ppf(3/4.), axis=0, center=np.median):
     return np.median((np.abs(a-center)) / c, axis=axis)
 
 
+def iqr(a, c=Gaussian.ppf(3/4) - Gaussian.ppf(1/4), axis=0, center=np.median):
+    # c \approx 1.349
+    """
+    The Interquartile range along given axis of an array
+
+    Parameters
+    ----------
+    a : array_like
+        Input array.
+    c : float, optional
+        The normalization constant, used to get consistent estimates of the
+        standard deviation at the normal distribution.  Defined as
+        scipy.stats.norm.ppf(3/4.) - scipy.stats.norm.ppf(1/4.), which is
+        approximately 1.349.
```

Comment:
This is the comment, so you can remove the comment.

Reply:
Do you mean remove the # c \approx 1.349 you pointed out above?

Reply:
Yes.

Reply:
Got it, fixed in commit ce3770f2774fe3c61fd7b1e5c1d4657d3731e1bf